dorasdb/CreateTraining.sql

```diff
@@ -3,5 +3,5 @@ CREATE TABLE Training(
     title varchar(50) NOT NULL default'Under work',
     trainingType varchar(255), /*Mandatory, Elective, Selective*/
     link varchar(89),
-    trainingDescription varchar(510) DEFAULT ' '
+    trainingDescription varchar(768) DEFAULT ' '
```

Comment:
any reason for that specific number?

Reply:
It's just a bigger number, some of the descriptions pulled from old ladder were too large to fit in 500 characters